adapter-s4j/src/main/java/io/nosqlbench/adapter/s4j/util/S4JClientConf.java

```diff
@@ -156,12 +158,9 @@ else if (StringUtils.startsWith(confKey, JMS_CONF_PREFIX)) {
                     }
                 }
             }
-        } catch (IOException ioe) {
-            logger.error("Can't read the specified config properties file: " + fileName);
-            ioe.printStackTrace();
-        } catch (ConfigurationException cex) {
-            logger.error("Error loading configuration items from the specified config properties file: " + fileName + ":" + cex.getMessage());
-            cex.printStackTrace();
+        } catch (IOException | ConfigurationException ex) {
```

Comment:
Is there an exception type leak here? In other words if an exception besides these two were to happen, is it handled as expected?

Reply:
I believe the code snippet above only throws out these 2 types of exceptions.